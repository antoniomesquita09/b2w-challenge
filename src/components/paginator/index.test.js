import React from 'react'
import { render } from '@testing-library/react'

import Paginator from './index.js'

describe('<Paginator />', () => {
  it('Snapshot testing', () => {
    const { asFragment } = render(<Paginator />)

    expect(asFragment()).toMatchSnapshot()
  })
})
