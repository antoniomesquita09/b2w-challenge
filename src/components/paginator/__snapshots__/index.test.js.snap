// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Paginator /> Snapshot testing 1`] = `
<DocumentFragment>
  <div
    class="root"
  />
</DocumentFragment>
`;
